### PR TITLE
fix: digital emoji unified handle.

### PR DESCRIPTION
**Pull request by web3max**

### CODE REVIEW

File: src/const.ts [hunk 1]
```diff
@@ -120,3 +120,26 @@ export const languageToCharType = {
 }
 
 export const languages = ['en', 'tr', 'vi', 'th', 'ko']
+
+export enum DigitalEmojiUnifiedMap {
+  '0⃣️' = '0️⃣',
```
Comment (zgayjjf) [inline src/const.ts:125]: Is the square(ZWS) necessary?

Comment (web3max) [reply src/const.ts:125]: Yes, read this document https://dasfoundation.larksuite.com/wiki/wikusrYVycLh2DXSjdwLpxjqOfh

File: src/tools/account.ts [hunk 2]
```diff
@@ -166,18 +184,26 @@ function getLanguage (language: string): string {
  */
 export function graphemesAccount (account: string, addSuffix = false, language = 'en'): ICharInfo[] {
   const splitter = new GraphemeSplitter()
-  const split = splitter.splitGraphemes(account)
+  let split = splitter.splitGraphemes(account)
+  split = split.map((item) => {
+    if (DigitalEmojiUnifiedMap[item]) {
```
Comment (zgayjjf) [inline src/tools/account.ts:184]: This can be simpler:`return DigitalEmojiUnifiedMap[item] || item`

File: tests/tools/account.test.ts [hunk 3]
```diff
@@ -180,3 +181,17 @@ describe('getAccountCharsetTypes', function () {
...
+describe('digitalEmojiUnifiedHandle', function () {
+  it('0⃣️.bit', () => {
+    expect(digitalEmojiUnifiedHandle('0⃣️.bit')).toEqual('0️⃣.bit')
+  })
+
+  it('1️⃣.bit', () => {
+    expect(digitalEmojiUnifiedHandle('1️⃣.bit')).toEqual('1️⃣.bit')
+  })
+
+  it('2⃣.bit', () => {
```
Comment (zgayjjf) [inline tests/tools/account.test.ts:194]: There can be another test, `imac0011️⃣.bit`, to prevent affecting the normal account.

**Review by zgayjjf — APPROVED** [review]

**Pull request merged**